hatch/src/dtl/tup/project.rs

```diff
@@ -20,20 +20,22 @@ impl ProjectAssets {
   pub fn config(project: &Project) -> ProjectAssets {
     let file_path = "C++/libs/".to_owned() + project.name() + "/config.tup";
 
-    let file_contents = match project.kind() {
-      &ProjectKind::Library(ref shared_lib) => format!( // {{{
-"PROJECT = {}
-LIB_TYPE = shared", // }}}
-      project.name()).to_string(),
-      
-      &ProjectKind::Library(ref static_lib) => format!( // {{{
+    let file_contents = match *project.kind() {
+      ProjectKind::Library(ref lib_type) => match lib_type {
+        &LibraryKind::Shared => format!( // {{{
+"PROJEct = {}
+LIB_TYpe = shared", // }}}
```

Comment:
typo?

Reply:
I wrote this awhile ago before I understood how match statements worked, originally this didnt match the lib type of the project type because it comes in as an associated type, now it does.